lib/bigstringaf.ml

```diff
@@ -83,33 +83,38 @@ let[@inline never] invalid_bounds_memcmp op buf1_len buf1_off buf2_len buf2_off
  * inequality did. As long as there is a check to ensure that [off] is not
  * negative, it won't underflow either. *)
 
-let copy t ~off ~len =
+let[@inlined] get_bounds name ?(off=0) ?len t =
```

Comment:
I think you want to annotate `inline` instead `inlined` (where the last one is about function application - to check if function is really _inlined_).

Reply:
I actually wanted to use ``@inlined``. According to the documentation it may also be used on functions, not only on function applications. Would you rather keep it as is, remove it or use ``@inline always`` ?

Reply:
Hmmhmm where you see that `inlined` can be use on a function declaration? I don't see that :s

Reply:
[8.13.1  Built-in attributes](https://caml.inria.fr/pub/docs/manual-ocaml-4.08/manual035.html#sec263)
> “ocaml.inlined” or “inlined” can be applied to any function or functor application to check that the call is inlined by the compiler. If the call is not inlined, a warning (55) is emitted. 

Oh, I see, "function or functor application", not "function or function application" :confused:

So replace it with ``@inline always`` or just throw it away?

Reply:
You can replace by `[@inline]` 😃